clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/LeaderFollowerPartitionStateModel.java

```diff
@@ -201,6 +192,27 @@ public void onBecomeOfflineFromError(Message message, NotificationContext contex
     logger.warn("unexpected state transition from ERROR to OFFLINE");
   }
 
+  private void updateLagMonitor(String resourceName, BiConsumer<Version, Integer> lagMonFunction) {
+    try {
+      String storeName = Version.parseStoreFromKafkaTopicName(resourceName);
+      int storeVersion = Version.parseVersionFromKafkaTopicName(resourceName);
+      Pair<Store, Version> res = getStoreRepo().waitVersion(storeName, storeVersion, Duration.ofMillis(500));
+      Store store = res.getFirst();
+      Version version = res.getSecond();
+      if (store == null || version == null) {
+        logger.warn(
```

Comment:
Let's bump this to error, this is pretty bad if we get here since we won't get metrics.

Reply:
done